changelogs/fragments/5280-lxc_container-py3.yaml

```
@@ -0,0 +1,5 @@
+bugfixes:
+  - The ``lxc_container``` module has been updated to support Python 3 (https://github.com/ansible-collections/community.general/pull/5304)
```

```suggestion
  - The ``lxc_container``` module has been updated to support Python 3 (https://github.com/ansible-collections/community.general/pull/5304). 
```

---

Please note that the module name must come first in the fragment, like
```suggestion
  - lxc_container - the module has been updated to support Python 3 (https://github.com/ansible-collections/community.general/pull/5304).
```
Same for the other fragment in this file.